Code/Editor/EditorPreferencesDialog.cpp

```diff
@@ -112,6 +112,24 @@ void EditorPreferencesDialog::showEvent(QShowEvent* event)
     QDialog::showEvent(event);
 }
 
+QList<const char*> list;
+
+void EditorPreferencesDialog::keyPressEvent([[maybe_unused]]QKeyEvent* event)
```

Comment:
I'm not entirely sure this is the proper way to handle the issue. The intention is to replace the preference system though, so I'd be ok with this if it's meant to be temporary.

Else, I think the input should consume the event and prevent it from being propagated out.

Reply:
keyPressEvent assumes it is consumed unless you explicitly call the parent.